rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -38,6 +38,7 @@
 
 
 PROTOCOLS_QUERY_NUM = 40  # number of protocols to query in a single call
+EXCLUDE_PROTOCOL_NAMES = {'Aave • Staking'}
```

Comment:
don't do it like this. We do have `DEFI_PROTOCOLS_TO_SKIP_ASSETS` where you should add it